build.sh

```diff
@@ -7,14 +7,27 @@ set -e
 DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
 cd $DIR
 
+x=0
+for arg in "$@"; do
+	let x=x+1
+	if test "$arg" = "--use-cache"; then
+		KAPACITOR_USE_BUILD_CACHE=defined
+		set -- "${@:1:$(expr $x - 1)}" "${@:$(expr $x + 1):$#}"
+		break
+	fi
+done
```

Comment:
Instead of arg parsing I would rather that the KAPACITOR_USE_BUILD_CACHE be set externally to the script if one wants to use it. While odd, this is currently the way behavior is determined in the script.

Something like:

`KAPACITOR_USE_BUILD_CACHE=${KAPACITOR_USE_BUILD_CACHE-false}`